src/collection/queries/get-collections-by-owner.graphql

```diff
@@ -68,5 +68,25 @@ query getCollectionsByOwner(
 				}
 			}
 		}
+		share_type
+		contributors {
+			enum_right_type {
+				value
+			}
+			profile {
+				usersByuserId {
```

Comment:
OPTIONAL: this relationship is named a bit funky in the database
we can add an alias to use user instead in the query
This allows us to use `contributor.profile.user` instead of `contributor.profile.usersByuserId` in the code

and that means you might be able to use Avo.User.Profile to type these fields instead of having to use custom types.
```suggestion
				user: usersByuserId {
```